src/components/RecipeView.vue

```diff
@@ -223,6 +239,20 @@ export default {
         }
     },
     methods: {
+        escapeHtml: function(unsafeString) {
+            return unsafeString
+                .replace(/&/g, "&amp;")
+                .replace(/\~/g, "&#732;")
+                .replace(/</g, "&lt;")
+                .replace(/>/g, "&gt;")
+                .replace(/"/g, "&quot;")
+                .replace(/'/g, "&#039;");
+        },
+        convertRecipeReferences: function(text) {
+            let re = /(?<=^|\s)#r\/(\d+)(?=$|\s)(\s|$)/g
```

Comment:
```suggestion
            let re = /(?<=^|\s|[,._+&?!-])#r\/(\d+)(?=$|\s|[.,_+&?!-])/g
```

This sounds still strange to me (a regular match after a lookahead does not make much sense to me). I find https://regex101.com very useful for testing. I added some punctuation before and after to allow matching with them adjacent as well.

Reply:
okay, your probably right. that’s probably just duplicating things. however one needs to edit the replacement statement as well, since no second captured group exists anymore.